README.rst

```diff
@@ -41,12 +41,14 @@ found at ``~/.config/mopidy/mopidy.conf``::
     [scrobbler]
     username = alice
     password = secret
+    backend_blacklist = list
```

Comment:
I think it's worth considering a word other than "blacklist". How about "ignorelist"?

Reply:
Agree.

We also have some "ignore"/"exclude" lists in the config of other extensions, like Mopidy-Local and Mopidy-Files. I'd like this to be aligned with the naming there.